recipes/mariadb-connector-cpp/all/conandata.yml

```
@@ -0,0 +1,8 @@
+sources:
+  "1.1.4":
+    url:
+      - "https://github.com/mariadb-corporation/mariadb-connector-cpp/archive/refs/tags/1.1.4.zip"
+    sha256: "4b6846abff09b9fece8c3766785248b60aa7e11815e5b4590bbf7dff7f2e7b3d"
+patches:
+  "1.1.4":
+    - patch_file: "patches/0001-1.1.4-cmake-install-logic.patch"
```

To finally fix all linter issues
```suggestion
    - patch_file: "patches/0001-1.1.4-cmake-install-logic.patch"
      patch_type: "conan"
      patch_description: "Compile both static and shared libraries but only install one of them"
```

---

Macos just can't find mariadbcpp-static.a, so I had to do this

---

but mariadbcpp-static.a exists